schema.graphql

```diff
@@ -253,12 +253,11 @@ type FpmmTransaction @entity {
   user: Account!
   fpmmType: FpmmType!
   transactionType: TransactionType!
-  collateralAmount: BigInt!
-  collateralAmountUSD: BigDecimal!
-  collateralTokenAddress: Bytes!
```

Comment:
Do we not need the collateral address for anything?

Reply:
yes for getting token but i can get that out of fpmm so there was no need for it